packages/features/tasker/tasks/crm/createCRMEvent.ts

```diff
@@ -0,0 +1,144 @@
+import type { Prisma } from "@prisma/client";
+
+import { eventTypeAppCardZod } from "@calcom/app-store/eventTypeAppCardZod";
+import logger from "@calcom/lib/logger";
+import prisma from "@calcom/prisma";
+import { BookingStatus } from "@calcom/prisma/enums";
+import { EventTypeMetaDataSchema } from "@calcom/prisma/zod-utils";
+
+import buildCalendarEvent from "./lib/buildCalendarEvent";
+import { createCRMEventSchema } from "./schema";
+
+const log = logger.getSubLogger({ prefix: [`[[tasker] createCRMEvent`] });
+
+export async function createCRMEvent(payload: string): Promise<void> {
```

Comment:
We should keep the original payload here, the parsing should happen on the task itself.

Reply:
I'm not sure what you mean here. The payload is just the `bookingUid`

Reply:
Mostly [this](https://github.com/calcom/cal.com/pull/18370/commits/a26100129eb7ea6f16a9f0440f099ed2cc5bbd25).